tools/RAiDER/aria/calcGUNW.py

```diff
@@ -88,8 +88,12 @@ def update_gunw_slc(path_gunw:str, ds_slc):
     with h5py.File(path_gunw, 'a') as h5:
         for k in TROPO_GROUP.split():
             h5 = h5[k]
-        del h5[TROPO_NAMES[0]]
-        del h5[TROPO_NAMES[1]]
+        # in case GUNW has already been updated once before
+        try:
+            del h5[TROPO_NAMES[0]]
+            del h5[TROPO_NAMES[1]]
+        except:
```

Comment:
Do not leave except statement without a specific error you expect. I think in this case you want KeyError.